app/api_alpha/views.py

```diff
@@ -513,9 +554,17 @@ class SingleBuilding(APIView):
         }
     )
     def get(self, request, rnb_id):
-        qs = list_bdgs({"user": request.user, "status": "all"}, only_active=False)
+
+        # check if we need to include plots
+        with_plots_param = request.query_params.get("withPlots", False)
+        with_plots = True if with_plots_param == "1" else False
```

Comment:
on pourrait écrire plus simplement 

```suggestion
        with_plots = with_plots_param == "1"
```

mais si `with_plots_params`devient lui même un booléen, ça va peut-être encore se simplifier !